easybuild/easyblocks/generic/binary.py

```diff
@@ -98,6 +98,11 @@ def install_step(self):
         """Copy all files in build directory to the install directory"""
         if self.cfg['install_cmd'] is None:
             try:
+                cmd = self.cfg['preinstallopts']
+                if cmd:
+                    self.log.info("Running preinstallopts of %s using command '%s'..." % (self.name, cmd))
+                    run_cmd(cmd, log_all=True, simple=True)
+                self.log.info("Copying %s into '%s'..." % (self.cfg['start_dir'], self.installdir))
```

Comment:
Yes, I know, normally you have something like:

```
cmd = ' '.join([self.cfg['preinstallopts'], self.cfg['install_cmd'], self.cfg['installopts']])
self.log.info("Installing %s using command '%s'..." % (self.name, cmd))
run_cmd(cmd, log_all=True, simple=True)
```

Which 90% of the time gets "abused" and becomes `foo &&` as you said. But regardless of if this is abuse or not, it should be consistent. Otherwise you have `preinstallopts` silently ignored.